src/nrniv/CMakeLists.txt

```diff
@@ -582,11 +582,13 @@ endif()
 # =============================================================================
 
 # For testneuron CTest
-file(COPY ${PROJECT_SOURCE_DIR}/src/ivoc/nrnmain.cpp DESTINATION ${CMAKE_BINARY_DIR}/share/nrn)
-file(COPY ${PROJECT_BINARY_DIR}/src/oc/nrnmpiuse.h DESTINATION ${CMAKE_BINARY_DIR}/include)
-file(COPY ${PROJECT_BINARY_DIR}/src/nrncvode/nrnneosm.h
-     DESTINATION ${CMAKE_BINARY_DIR}/include/nrncvode)
-file(COPY ${PROJECT_BINARY_DIR}/nrnconf.h DESTINATION ${CMAKE_BINARY_DIR}/include)
+cpp_cc_build_time_copy(INPUT ${PROJECT_SOURCE_DIR}/src/ivoc/nrnmain.cpp
+                       OUTPUT ${CMAKE_BINARY_DIR}/share/nrn)
+cpp_cc_build_time_copy(INPUT ${PROJECT_BINARY_DIR}/src/oc/nrnmpiuse.h
+                       OUTPUT ${CMAKE_BINARY_DIR}/include)
+cpp_cc_build_time_copy(INPUT ${PROJECT_BINARY_DIR}/src/nrncvode/nrnneosm.h
+                       OUTPUT ${CMAKE_BINARY_DIR}/include/nrncvode)
+cpp_cc_build_time_copy(INPUT ${PROJECT_BINARY_DIR}/nrnconf.h OUTPUT ${CMAKE_BINARY_DIR}/include)
```

Comment:
For some reason you seem to be getting a cmake configure error here
https://dev.azure.com/neuronsimulator/nrn/_build/results?buildId=9631&view=logs&jobId=13914a51-6bc1-5021-d84a-2ddaccaa6076&j=13914a51-6bc1-5021-d84a-2ddaccaa6076&t=f9dcca68-16a5-52cb-8082-cf6f597bb4c5